src/ansys/mapdl/core/mapdl.py

```diff
@@ -460,15 +460,123 @@ def _chain_stored(self):
+        to zero (i.e. ``array40``) and another one with the second axis equal
+        to one, if later retrieved, they will have the same
+        shape.
+
+        >>> array41 = np.reshape([1,2,3, 4], (4,1))
+        >>> array41
+        array([[1],
+        [2],
+        [3],
+        [4]])
+        >>> mapdl.parameters['mapdlarray41'] = array41
+        >>> mapdl.parameters['mapdlarray41']
+        array([[1.],
+        [2.],
+        [3.],
+        [4.]])
+        >>> np.allclose(mapdl.parameters['mapdlarray40'] == assert mapdl.parameters['mapdlarray41'])
+        True
+
+        **Leading underscored parameters**
+
+        The parameters starting with underscore ('_') are reserved parameters
+        for MAPDL macros and routines, therefore its use is discouraged, and
+        in PyMAPDL you cannot set them by default.
+
+        If you need to set one of this parameters, you can use ``mapdl._run``
+        to avoid PyMAPDL parameter name checks. For example
+
+        >>> mapdl._run('_parameter=123')
+        'PARAMETER _PARAMETER =     123.00000000'
+
+        By default, this type of parameters cannot be seen when issuing
+        ``mapdl.parameters``. However, you can change this by setting
+        ``mapdl.parameters.show_leading_underscore_parameters`` equal to True.
+        For example:
+
+        >>> mapdl.parameters.show_leading_underscore_parameters=True
+        >>> mapdl.parameters
+        MAPDL Parameters
+        ----------------
+        PORT                             : 50053.0
+        _RETURN                          : 0.0
+        _STATUS                          : 0.0
+        _UIQR                            : 17.0
+
+
+        **Trailing underscored parameters**
+
+        The parameters ending underscore are recommend for user routines
+        and macros.
+        You can set this type of parameters in PyMAPDL, but by default,
+        they cannot be seen in ``mapdl.parameters``, unless
+        ``mapdl.parameters.show_trailing_underscore_parameters`` is set
+        to True.
+
+        >>> mapdl.parameters['param_'] = 1.0
+        >>> mapdl.parameters
+        MAPDL Parameters
+        ----------------
+        >>> mapdl.parameters.show_trailing_underscore_parameters=True
+        >>> mapdl.parameters
+        MAPDL Parameters
+        ----------------
+        PARAM_                           : 1.0
+
+        **Parameters with leading and trailing underscore**
+
+        These are an especial type of parameters. They CANNOT be seen
+        in ``mapdl.parameters`` under any circumstances, and because
+        of it, it uses is not recommended.
+
+        You can still retrieve them using any of the normal methods
+        to retrieve parameters. But you shall know the parameter name.
+        For example:
+
+        >>> mapdl.parameters["_param_"] = 1.0
+        >>> mapdl.parameters
+        MAPDL Parameters
+        ----------------
+        >>> print(mapdl.parameters['_param_'])
+        1.0
```

Comment:
Follow the [DRY Principal](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

```suggestion
        See :ref:`special_named_param` for additional notes regarding parameter naming in MAPDL.
```